src/app/screens/createInscription/index.tsx

```diff
@@ -0,0 +1,479 @@
+      const response = {
+        source: MESSAGE_SOURCE,
+        method: ExternalSatsMethods.createInscriptionResponse,
+        payload: {
+          createInscriptionRequest: requestToken,
+          createInscriptionResponse: {
+            txId: revealTransactionId,
+          },
+        },
+      };
+      chrome.tabs.sendMessage(tabId, response);
+      window.close();
+    };
+
+    return <CompleteScreen txId={revealTransactionId} onClose={onClose} network={network} />;
+  }
+
+  const errorCode = feeErrorCode || executeErrorCode;
+
+  return (
+    <ConfirmScreen
+      onConfirm={executeMint}
+      onCancel={cancelCallback}
+      cancelText={t('CANCEL_BUTTON')}
+      confirmText={!errorCode ? t('CONFIRM_BUTTON') : t(`ERRORS.SHORT.${errorCode}`)}
+      loading={isExecuting || isLoading}
+      disabled={!!errorCode || isExecuting}
+      isError={!!errorCode}
+    >
+      <OuterContainer>
+        <AccountHeaderComponent disableMenuOption disableAccountSwitch disableCopy />
+        <MainContainer>
+          <Title>{t('TITLE')}</Title>
+          <SubTitle>{t('SUBTITLE')}</SubTitle>
+          <CardContainer bottomPadding>
+            <CardRow>
+              <div>{t('SUMMARY.TITLE')}</div>
+            </CardRow>
+            <CardRow topMargin center>
+              <IconLabel>
+                <div>
+                  <ButtonIcon src={OrdinalsIcon} />
+                </div>
+                <div>{t('SUMMARY.ORDINAL')}</div>
+              </IconLabel>
+              <ContentLabel contentType={contentType} content={content} type={payloadType} />
+            </CardRow>
+            <CardRow topMargin>
+              <MutedLabel>{t('SUMMARY.TO')}</MutedLabel>
+            </CardRow>
+            <CardRow topMargin center>
+              <IconLabel>
+                <InfoIconContainer>
+                  <InfoIcon src={WalletIcon} />
+                </InfoIconContainer>
+                {t('SUMMARY.YOUR_ADDRESS')}
+              </IconLabel>
+              <div>{getShortTruncatedAddress(ordinalsAddress)}</div>
+            </CardRow>
+          </CardContainer>
+          <CardContainer>
+            <CardRow>
+              <div>{t('NETWORK')}</div>
+              <div>{network.type}</div>
+            </CardRow>
+          </CardContainer>
+          <CardContainer>
+            <CardRow>
+              <div>{t('VALUE')}</div>
+              <div>
+                {isLoading && <MoonLoader color="white" size={20} />}
+                {!isLoading && (
+                  <NumberWithSuffixContainer>
+                    <NumericFormat
+                      value={commitValue}
+                      displayType="text"
+                      thousandSeparator
+                      suffix=" sats"
+                    />
+                    <NumericFormat
+                      value={fiatValue}
+                      displayType="text"
+                      thousandSeparator
+                      suffix=" USD"
```

Comment:
This should not be hardcoded I think, but rather picked up from selected fiat currency. 
`const { fiatCurrency } = useWalletSelector();`

Reply:
Ah, woops! Thanks for catching this 🙏